features/mgmt_api/steps/steps.py

```diff
@@ -9,3 +19,47 @@ def given_empty_auth_token(context):
         context.headers = []
 
     context.headers = [header for header in context.headers if header[0] != "Authorization"]
+
+
+async def store_placeholder(context, url):
```

Comment:
would be good to add some docstrings explaining the "why" of this method. Plus, types won't hurt either :)